src/storm/settings/modules/MinMaxEquationSolverSettings.cpp

```diff
@@ -64,8 +64,8 @@ MinMaxEquationSolverSettings::MinMaxEquationSolverSettings() : ModuleSettings(mo
                                          .build())
                         .build());
 
-    this->addOption(storm::settings::OptionBuilder(moduleName, intervalIterationSymmetricUpdatesOptionName, false,
-                                                   "If set, interval iteration performs an update on both, lower and upper bound in each iteration")
+    this->addOption(storm::settings::OptionBuilder(moduleName, forceUniqueSolutionRequirementOptionName, false,
+                                                   "If set, a MinMax solver always requires a unique solution for its input.")
```

Comment:
I find this a very vague description for a user setting. Can we provide some additional context when a user should set this flag?

Reply:
True, what about this:
`Enforces end component collapsing for MinMax equation systems so that their solution becomes unique. May simplify solving but causes some overhead` ?